### ISSUE TITLE
The serial number of some newly booted nodes on testnet is displayed as `Default string`

### ISSUE THREAD

**Issue by mohamedamer453** [threefoldtech/tfgrid_dashboard#603]
### Description

The serial number of some newly booted nodes on testnet is displayed as `Default string`.

![image](https://user-images.githubusercontent.com/101194226/228820016-f3cb712a-f543-4dc4-9d52-d6555f7efde5.png)

### Environment

- Network: `testnet`.
- Dashboard Version: `v1.4.0-rc10`

**Comment by ashraffouda** [comment]
this value only exists in dmi_test.go in zos https://github.com/threefoldtech/zos/blob/main/pkg/capacity/dmi/dmi_test.go#L877
so maybe those nodes were created as a part of tests and not cleaned up?

**Comment by mohamedamer453** [comment]
> this value only exists in dmi_test.go in zos https://github.com/threefoldtech/zos/blob/main/pkg/capacity/dmi/dmi_test.go#L877 so maybe those nodes were created as a part of tests and not cleaned up?

@coesensbert These are the testnet nodes that were booted last week to test the famerbot. were they used before as testing nodes and not cleaned up as mentioned above?

**Comment by coesensbert** [comment]
> > this value only exists in dmi_test.go in zos https://github.com/threefoldtech/zos/blob/main/pkg/capacity/dmi/dmi_test.go#L877 so maybe those nodes were created as a part of tests and not cleaned up?
> 
> @coesensbert These are the testnet nodes that were booted last week to test the famerbot. were they used before as testing nodes and not cleaned up as mentioned above?

Yes indeed, they are 4 silver boxes that where used to test WOL on devnet.
We did clean both disks (ssd and hdd) before changing nets with `wipefs -a /dev/sdX` so they should have been started clean.

I can't explain the 'Default string' but it might be normal since we have some of those silverboxes on mainnet also, with the same serial nr. These are from farm Silverfarm with ID 3 on mainnet:
![image](https://user-images.githubusercontent.com/88312671/229449839-05b44ee0-3fee-4df8-bcbc-22eaec39c5fd.png)

**Comment by muhamadazmy** [comment]
I think dmidecode itself return `Default string`. It's not a dmidecode issue it's the node hardware itself has this as a serial number burned in the chip!

in other words that's literaly what is stored on the board as `serial number` and in that case we can just mask them out. (replace Default string with `<noset>` or something

**Comment by muhamadazmy** [comment]
The silver boxes always had this as the `serial number` . i think it's just lazy manufacturer
